iphone/Maps/Core/Settings/MWMSettings.mm

```diff
@@ -167,4 +168,14 @@ + (void)setICLoudSynchronizationEnabled:(BOOL)iCLoudSyncEnabled
   [NSUserDefaults.standardUserDefaults setBool:iCLoudSyncEnabled forKey:kiCLoudSynchronizationEnabledKey];
   [NSNotificationCenter.defaultCenter postNotificationName:NSNotification.iCloudSynchronizationDidChangeEnabledState object:nil];
 }
+
++ (BOOL)isFileLoggingEnabled {
```

Comment:
Can this one be used instead of static BOOL?

Reply:
In objc the `class varibles` can be stored only: getter method + setter method + static variable to actually store the value.